Resources/Maps/_Mono/Shuttles/framework.yml

```
@@ -0,0 +1,564 @@
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+        - volume: 2500
+          immutable: True
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)